packages/@ourworldindata/grapher/src/controls/ContentSwitchers.tsx

```diff
@@ -151,7 +151,8 @@ export class ContentSwitchers extends React.Component<{
 
     @action.bound private onTabChange(selectedKey: TabKey): void {
         if (selectedKey === OVERFLOW_MENU_KEY) {
-            this.showOverflowMenu()
+            // Prevent the click from immediately closing the popover
+            setTimeout(() => this.showOverflowMenu(), 0)
```

Comment:
I'll get rid of this in the next PR when I rewrite the Popover component to use react-aria